app/gui2/src/components/GraphEditor/clipboard.ts

```diff
@@ -22,16 +22,14 @@ interface ClipboardData {
 interface CopiedNode {
   expression: string
   documentation?: string | undefined
-  visualization?: VisualizationMetadata | undefined
-  colorOverride?: string | undefined
+  metadata?: NodeMetadataFields
```

Comment:
So we don't want to exclude position from copied metadata?

Reply:
`position` is ignored in the metadata supplied for node creation because a separate parameter overrides it.